<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8"/>
		<title>text-extractor</title>
		<!-- 		<link rel="stylesheet" type="text/css" href="style/style-1.0.css"/> -->
		<script type="text/javascript" src="script/jquery-1.11.0.js"></script>
		<script type="text/javascript" src="https://c328740.ssl.cf1.rackcdn.com/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
		<script type="text/javascript" src="script/textract-1.0.js"></script>
	</head>
	<body>
		<div>
			<header class="so-article-header">
				<div class="so-d so-article-header-text">
					<h2 class="so-article-header-title">On primes of the form \(n_1^u + n_2^v + k\), on average</h2>
					<h2 class="so-article-header-title">A BAC end view of the <i>Musa acuminata </i>genome</h2>
					<p class="so-article-header-info"><span class="so-metadata-label">
									Authors:
								</span><a id="A1" class="named-anchor"><!-- named anchor --></a>Foo Cheung<a href="#I1" class="author-link"><sup>1</sup></a>, <a id="A2" class="named-anchor"><!-- named anchor --></a>Christopher D Town<a href="#I1" class="author-link"><sup>1</sup></a></p>
					<p class="so-article-header-info"><span class="so-metadata-label">Publication date (<span class="data">Electronic</span>):</span> 11 June 2007</p>
					<p class="so-article-header-info"><span class="so-metadata-label">Source: PMC ID: </span>1904220</p>
					<p class="so-article-header-info"><span class="so-metadata-label">Journal: </span>BMC Plant Biology</p>
					<p class="so-article-header-info"><span class="so-metadata-label">Publisher: </span>BioMed Central</p>
					<p class="so-article-header-info"><span class="so-metadata-label">License: </span>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a target="xrefwindow" href="http://creativecommons.org/licenses/by/2.0" data-untrusted="" id="d4689524e105">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
					<p class="so-article-header-info"><span class="so-metadata-label">DOI: </span><a href="http://dx.doi.org/10.1186/1471-2229-7-29" data-untrusted="">10.1186/1471-2229-7-29</a></p>
				</div>
			</header>
			<section class="so-layout-section so-section-collapser" id="author-and-article-info">
				<header class="so-layout-section-header so-openable so-closed">
					<h3 class="so-layout-section-title">Author and article information</h3>
				</header>
				<div class="so-d">
					<div class="so-affiliations">
						<h6>
							<p>
								Affiliations
							</p>
						</h6>
						<div class="so-info-entry"><a id="I1" class="named-anchor"><!-- named anchor --></a><span class="generated">[</span>1<span class="generated">] </span>J Craig Venter Institute, 9712 Medical Center Drive, Rockville, MD 20850 USA</div>
					</div>
					<h6>
						<p>
							Contributors
						</p>
					</h6>
					<div class="so-info-enty"><span class="so-metadata-label">Foo Cheung </span></div>
					<div class="so-info-enty"><span class="so-metadata-label">Christopher D Town </span></div>
					<h6>
						<p>
							Journal
						</p>
					</h6>
					<div class="so-info-enty"><span class="so-metadata-label">Journal ID (nlm-ta): </span>BMC Plant Biol</div>
					<div class="so-info-enty"><span class="so-metadata-label">Title: </span>BMC Plant Biology</div>
					<div class="so-info-enty"><span class="so-metadata-label">Publisher: </span>BioMed Central<span class="generated"> (</span>London<span class="generated">)</span></div>
					<div class="so-info-enty"><span class="so-metadata-label">ISSN (Electronic): </span>1471-2229</div>
					<div class="so-info-entry">
						<div class="so-info-enty"><span class="so-metadata-label">Publication date (Collection): </span>2007</div>
					</div>
					<div class="so-info-entry">
						<div class="so-info-enty"><span class="so-metadata-label">Publication date (Electronic): </span>11 June 2007</div>
					</div>
					<div class="so-info-enty"><span class="so-metadata-label">Copyright statement: </span>Copyright © 2007 Cheung and Town; licensee BioMed Central Ltd.</div>
					<h6>
						<p>
							Categories
						</p>
					</h6>
					<div class="so-info-enty"><span class="so-metadata-label">Subject: </span>Research Article</div>
				</div>
			</section>
			<section class="so-layout-section so-section-collapser" id="d4689524e107">
				<header class="so-layout-section-header so-openable so-open" data-is-open="">
					<h3 class="so-layout-section-title">Abstract</h3>
				</header>
				<div class="so-d">
					<div class="section"><a id="d4689524e108" class="named-anchor"><!-- named anchor --></a>
						<h5 class="section-title">Background</h5>
						<p id="d4689524e111"><i>Musa </i>species contain the fourth most important crop in developing countries. Here, we report the analysis of 6,252 BAC end-sequences, in order to view the sequence composition of the <i>Musa acuminata </i>genome in a cost effective and efficient manner.</p>
					</div>
					<div class="section"><a id="d4689524e118" class="named-anchor"><!-- named anchor --></a>
						<h5 class="section-title">Results</h5>
						<p id="d4689524e121">BAC end sequencing generated 6,252 reads representing 4,420,944 bp, including 2,979 clone pairs with an average read length after cleaning and filtering of 707 bp. All sequences have been submitted to GenBank, with the accession numbers <a target="xrefwindow" href="DX451975" data-untrusted="" id="d4689524e123">DX451975</a> – <a target="xrefwindow" href="DX458350" data-untrusted="" id="d4689524e126">DX458350</a>. The BAC end-sequences, were searched against several databases and significant homology was found to mitochondria and chloroplast (2.6%), transposons and repetitive sequences (36%) and proteins (11%). Functional interpretation of the protein matches was carried out by Gene Ontology assignments from matches to <i>Arabidopsis </i>and was shown to cover a broad range of categories. From protein matching regions of <i>Musa </i>BAC end-sequences, it was determined that the GC content of coding regions was 47%. Where protein matches encompassed a start codon, GC content as a function of position (5' to 3') across 129 bp sliding windows generates a "rice-like" gradient. A total of 352 potential SSR markers were discovered. The most abundant simple sequence repeats in four size categories were AT-rich. After filtering mitochondria and chloroplast matches, thousands of BAC end-sequences had a significant BLASTN match to the <i>Oryza sativa </i>and <i>Arabidopsis </i>genome sequence. Of these, a small number of BAC end-sequence pairs were shown to map to neighboring regions of the <i>Oryza sativa </i>genome representing regions of potential microsynteny.</p>
					</div>
					<div class="section"><a id="d4689524e145" class="named-anchor"><!-- named anchor --></a>
						<h5 class="section-title">Conclusion</h5>
						<p id="d4689524e148">Database searches with the BAC end-sequences and <i>ab initio </i>analysis identified those reads likely to contain transposons, repeat sequences, proteins and simple sequence repeats. Approximately 600 BAC end-sequences contained protein sequences that were not found in the existing available <i>Musa </i>expressed sequence tags, repeat or transposon databases. In addition, gene statistics, GC content and profile could also be estimated based on the region matching the top protein hit. A small number of BAC end pair sequences can be mapped to neighboring regions of the <i>Oryza sativa </i>representing regions of potential microsynteny. These results suggest that a large-scale BAC end sequencing strategy has the potential to anchor a small proportion of the genome of <i>Musa acuminata </i>to the genomes of <i>Oryza sativa </i>and possibly <i>Arabidopsis</i>.</p>
					</div>
				</div>
			</section>
			<section class="so-layout-section so-section-collapser" id="main-article-text">
				<header class="so-layout-section-header so-openable so-open">
					<h3 class="so-layout-section-title">Main article text</h3>
				</header>
				<div class="so-d" style="display: block;">
					<div class="section"><a id="d4689524e170" class="named-anchor"><!-- named anchor --></a>
						<h4 class="main-title">Background</h4>
						<p id="d4689524e173">Until novel technologies that will enable extremely low-cost genomic DNA sequencing are developed, funding bodies are very selective when choosing new plant genomes to sequence. Current technologies are only able to produce the sequence of a mammalian-sized genome of the desired data quality for $10 to $50 million or more. The initial goal of many genome projects is often to gain a glimpse of the genome of interest at a low cost and in an effective manner. In plants there is often some advantage in leveraging the finished genomes of <i>Arabidopsis thaliana </i>and <i>Oryza sativa </i>through comparative genomics. <i>A. thaliana </i>was chosen as model for the dicotyledons due to its small genome size (125 Mb) [<a href="#B1" class="xref-link">1</a>] and rice [<a href="#B2" class="xref-link">2</a>] (<i>O. sativa</i>) was the first cereal and monocot to be sequenced [<a href="#B3" class="xref-link">3</a>].</p>
						<p id="d4689524e197"><i>Musa </i>species (bananas and plantains) comprise very important crops in sub-Saharan Africa, South and Central America and much of Asia. The <i>Musa </i>species <i>Musa acuminata </i>(AA genome) and <i>Musa balbisiana</i>, (BB genome), both with 2n = 22 chromosomes represent the two main progenitors of cultivated banana varieties. The haploid genome of <i>Musa </i>species was estimated as varying between 560 to 800 Mb in size [<a href="#B4" class="xref-link">4</a>-<a href="#B6" class="xref-link">6</a>], over four times larger than that of the model plant <i>A. thaliana </i>(125 Mb) [<a href="#B7" class="xref-link">7</a>] and over 30% larger than that of <i>O. sativa </i>(390 Mb) [<a href="#B2" class="xref-link">2</a>].</p>
						<p id="d4689524e232">Comparative genomics in the monocots have focused on the extent of synteny between closely-related species of monocots belonging to the family of Poaceae [<a href="#B8" class="xref-link">8</a>]. Extensive micro and macro synteny has been shown between <i>O. sativa</i>, barley, maize and wheat [<a href="#B9" class="xref-link">9</a>,<a href="#B10" class="xref-link">10</a>] and the degree of conservation often varies between different chromosomal locations. Synteny between distantly related plants is more bioinformatically challenging to elucidate and probably occurs less frequently.</p>
						<p id="d4689524e246">In order to understand the sequence content and sequence complexity of the <i>Musa </i>genome, it is necessary to sequence a large number of randomly selected clones that are representative of the entire genome. An alternative approach is to end-sequence a large number of Bacterial Artificial Chromosomes (BACs) randomly selected from a BAC library [<a href="#B11" class="xref-link">11</a>]. This latter approach does not provide a truly random sampling of the genome since regions in which the restriction site for the particular enzyme used for library construction is under-represented will also be under-represented. Nevertheless, BAC end sequencing does provide a quasi-random sampling of the genome and carries with it the advantage that BAC clones that appear to contain targets of interest provide excellent material for other analyses such as fluorescent <i>in situ </i>hybridization (FISH) to metaphase or pachytene chromosomes or in depth sequencing for gene discovery. A large collection of BAC end-sequences (BES) is also an essential component of a genome sequencing project. Here, we examined whether <i>Musa </i>BES can lead to insights into the <i>Musa </i>genome composition using bioinformatic comparisons to protein, repeat, expressed sequence tags (ESTs) and other databases. From the BES, we investigate the <i>Musa </i>gene density, GC content, protein and SSR content and putative comparative-tile BACs that represents potential regions of microsynteny between the <i>O. sativa </i>and <i>Musa </i>species.</p>
					</div>
					<div class="section"><a id="d4689524e273" class="named-anchor"><!-- named anchor --></a>
						<h4 class="main-title">Results and discussion</h4>
						<p id="d4689524e276">Sequence searches, simple sequence repeats, GC profiling and protein discovery will be discussed first, followed by an analysis of genome mapping to <i>O.sativa </i>and <i>A. thaliana </i>to identify comparative tile BACs from the <i>Musa </i>library that will be likely collinear (i.e. showed microsynteny).</p>
						<div class="section"><a id="d4689524e287" class="named-anchor"><!-- named anchor --></a>
							<h5 class="section-title">BAC end sequencing</h5>
							<p id="d4689524e290">End sequencing of BACs from a HindIII BAC library constructed from leaves of the wild diploid 'Calcutta 4' clone [<a href="#B12" class="xref-link">12</a>], generated 6,252 high quality reads with an average length of 707 nucleotides, giving a total length of ~ 4.4 Mb that included 2,979 paired end reads (Table <a href="#T1" class="xref-link">1</a>). All sequences have been submitted to GenBank, with the accession numbers <a target="xrefwindow" href="DX451975" data-untrusted="" id="d4689524e298">DX451975</a> – <a target="xrefwindow" href="DX458350" data-untrusted="" id="d4689524e301">DX458350</a>.</p>
							<div class="table-wrap panel"><a id="T1" class="named-anchor"><!-- named anchor --></a>
								<div class="panel-content">
									<div class="label">Table&nbsp;1</div>
									<div class="caption">
										<p class="first" id="d4689524e308">Sequence statistics of the <i>Musa </i>BES</p>
									</div>
									<div class="table-container">
										<table class="table-figure">
											<tbody>
												<tr>
													<td rowspan="1" colspan="1">Total # sequences</td>
													<td rowspan="1" colspan="1">6,252</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">Total base count (bp)</td>
													<td rowspan="1" colspan="1">4,420,944</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">Minimum length (bp)</td>
													<td rowspan="1" colspan="1">101</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">Average length (bp)</td>
													<td rowspan="1" colspan="1">707</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">Maximum length (bp)</td>
													<td rowspan="1" colspan="1">1,007</td>
												</tr>
											</tbody>
										</table>
									</div>
								</div>
							</div>
						</div>
						<div class="section"><a id="d4689524e340" class="named-anchor"><!-- named anchor --></a>
							<h5 class="section-title">Database sequence searches</h5>
							<p id="d4689524e343">Comparison of the BES with the TIGR non-identical amino acid database revealed that 11% of the sequences contained "genic" regions by virtue of good matches, excluding transposons/repeats (36%). Using a stringent threshold of 1e-5, 80% identity and 80% coverage resulted in 2.6% BES matches to chloroplast/mitochondria (Table <a href="#T2" class="xref-link">2</a>). Of the protein matches, the top BLAST match in over 50% of cases was to <i>O.sativa </i>and in 30% to <i>A. thaliana </i>proteins, consistent with the closer relatedness between <i>Musa </i>and <i>O. sativa </i>when compared to <i>Musa </i>and <i>A. thaliana</i>. This is also consistent with matches to the TIGR Plant Gene Indices where the highest level of homology was shown to <i>O. sativa </i>followed by barley, wheat and other monocots (Figure <a href="#F1" class="xref-link">1</a>). Of the BES analysed, 36% were found to contain sequences homologous to transposable elements or repeats. The majority of transposable elements belonged to the Ty1 copia type (742) followed by the Ty3 gypsy (211) types of retrotransposons (Table <a href="#T2" class="xref-link">2</a>) consistent with previous data that class I retrotransposons contributing to most of the nucleotide [<a href="#B13" class="xref-link">13</a>] and from studies using papaya BAC end sequences.</p>
							<div class="table-wrap panel"><a id="T2" class="named-anchor"><!-- named anchor --></a>
								<div class="panel-content">
									<div class="label">Table&nbsp;2</div>
									<div class="caption">
										<p class="first" id="d4689524e383">Sequence similarity search results</p>
									</div>
									<div class="table-container">
										<table class="table-figure">
											<thead>
												<tr>
													<td rowspan="1" colspan="1">Database</td>
													<td rowspan="1" colspan="1">Number of hits (%)</td>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td rowspan="1" colspan="1">Mitochondria + Chloroplast</td>
													<td rowspan="1" colspan="1">162 (2.6)</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">Transposon + Repeats</td>
													<td rowspan="1" colspan="1">2,291 (36.6)</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">TIGR protein database</td>
													<td rowspan="1" colspan="1">686 (11)</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1"><b>Total number of BAC ends</b></td>
													<td rowspan="1" colspan="1"><b>3,139 (50.2)</b></td>
												</tr>
											</tbody>
										</table>
									</div>
								</div>
							</div>
							<div class="fig panel"><a id="F1" class="named-anchor"><!-- named anchor --></a>
								<div class="figure-container so-text-align-c"><img class="figure" alt="" src="content/1.jpg"></div>
								<div class="panel-content">
									<div class="label">Figure&nbsp;1</div>
									<div class="caption">
										<p class="first" id="d4689524e419">Number of <i>Musa </i>BES containing hits to The TIGR Plant Gene Indices using blat.</p>
									</div>
								</div>
							</div>
							<p id="d4689524e425">We also found 111 matches to miniature inverted repeat transposable elements (MITEs), the most abundant being adh-11-like (46), followed by adh type D-like (22) and adh type G-like (12). Gene density predictions calculated from the number BES with protein matches (686) at E = 1e-15 estimates the presence of a gene every 6.4 kb (Table <a href="#T3" class="xref-link">3</a>) which is consistent with previous gene density studies from one <i>Musa </i>BAC studied [<a href="#B14" class="xref-link">14</a>]. In contrast, a second BAC from the same study gave a gene density of a gene in every 10 kb, however upon closer examination one half of the BAC consisted of transposon related genes while the other half was non-transposon related. The discrepancy between the data suggests that the gene organization resembling Gramineae where genes are clustered in gene-rich regions separated by gene-poor DNA containing abundant transposons. In comparison with other plant genomes, gene density appears to be similar to reports for the automatic annotation for <i>O. sativa </i>of 6.2 kb per gene [<a href="#B15" class="xref-link">15</a>] and different from <i>A. thaliana </i>with 4.5 kb per gene [<a href="#B6" class="xref-link">6</a>].</p>
							<div class="table-wrap panel"><a id="T3" class="named-anchor"><!-- named anchor --></a>
								<div class="panel-content">
									<div class="label">Table&nbsp;3</div>
									<div class="caption">
										<p class="first" id="d4689524e453">Summary of transposon content</p>
									</div>
									<div class="table-container">
										<table class="table-figure">
											<thead>
												<tr>
													<td rowspan="1" colspan="1">Transposon Type</td>
													<td rowspan="1" colspan="1">Number of BES</td>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td rowspan="1" colspan="1">Ty3-gypsy</td>
													<td rowspan="1" colspan="1">211</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">Ty1-copia</td>
													<td rowspan="1" colspan="1">742</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">LINE element</td>
													<td rowspan="1" colspan="1">12</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">MUDR element</td>
													<td rowspan="1" colspan="1">2</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">Athila</td>
													<td rowspan="1" colspan="1">4</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">Endovir</td>
													<td rowspan="1" colspan="1">31</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">MITES</td>
													<td rowspan="1" colspan="1">111</td>
												</tr>
											</tbody>
										</table>
									</div>
								</div>
							</div>
						</div>
						<div class="section"><a id="d4689524e498" class="named-anchor"><!-- named anchor --></a>
							<h5 class="section-title">Functional annotation</h5>
							<p id="d4689524e501">Gene Ontology (GO) is a controlled vocabulary of functional terms that allows consistent annotation of gene products [<a href="#B16" class="xref-link">16</a>]. In order to assign putative functional roles to the <i>Musa acuminata </i>sequences, we used the GO assignments of the <i>A. thaliana </i>proteome [<a href="#B16" class="xref-link">16</a>]. Among the 686 BES that did not contain a match to the repeat or transposon databases but contained a match the TIGR comprehensive protein database, 664 had matches to <i>A. thaliana </i>proteins and were given GO assignments based on the top matches. The genes are shown to cover a broad range of GO categories (Figure <a href="#F3" class="xref-link">3</a>).</p>
							<div class="fig panel"><a id="F3" class="named-anchor"><!-- named anchor --></a>
								<div class="figure-container so-text-align-c"><img class="figure" alt="" src="content/3.jpg"></div>
								<div class="panel-content">
									<div class="label">Figure&nbsp;3</div>
									<div class="caption">
										<p class="first" id="d4689524e526">Mean GC content as a function of position (5' to 3') across 129 bp sliding windows.</p>
									</div>
								</div>
							</div>
						</div>
						<div class="section"><a id="d4689524e529" class="named-anchor"><!-- named anchor --></a>
							<h5 class="section-title">GC profile</h5>
							<p id="d4689524e532">GC profiling was performed on the matching region between the BES and the top protein hit. Any BES not containing a match from the start codon was excluded. In parallel, a similar study was carried out for <i>A. thaliana</i>, <i>O. sativa</i>, maize and <i>Medicago truncatula </i>BES (Figure <a href="#F2" class="xref-link">2</a>). <i>A. thaliana </i>and <i>M. truncatula </i>showed similar GC content along the entire coding sequence. In most cases <i>Musa</i>, <i>O. sativa </i>and maize showed a higher GC value at the 5' end within the first 150 bp from the predicted start site, which gradually decreased towards the 3' end. This result is consistent from previous reports where it has been shown that grasses have high mean GC content and asymmetrical distributions, while the eudicots have lower GC content and more symmetrical distributions [<a href="#B17" class="xref-link">17</a>,<a href="#B18" class="xref-link">18</a>]</p>
							<div class="fig panel"><a id="F2" class="named-anchor"><!-- named anchor --></a>
								<div class="figure-container so-text-align-c"><img class="figure" alt="" src="content/2.jpg"></div>
								<div class="panel-content">
									<div class="label">Figure&nbsp;2</div>
									<div class="caption">
										<p class="first" id="d4689524e569">Gene Ontology assignments for <i>Musa </i>BES.</p>
									</div>
								</div>
							</div>
						</div>
						<div class="section"><a id="d4689524e575" class="named-anchor"><!-- named anchor --></a>
							<h5 class="section-title">GC content</h5>
							<p id="d4689524e578">The GC content for organisms varies between the genomic, intron and exon regions and can be as low as 22% (<i>Plasmodium falciparum</i>) to more than 70% (<i>Zea mays</i>). GC content was determined on the matching region between the BES and the top protein hit. The mean GC content of all BES was 39% and coding sequence GC content was 47% consistent with previous studies which was shown to have an overall GC content to be 38% and within exons to be 49% based on 2 BACs [<a href="#B14" class="xref-link">14</a>]. This and the previous section have shown that BES with protein matches can allow GC content and GC profiling to be calculated with some degree of accuracy. Further confirmation using a larger dataset was carried out using ESTs,- 2,280 <i>Musa </i>ESTs [<a href="#B19" class="xref-link">19</a>] was downloaded from GenBank, clustered and assembled to give 1,123 unique sequences of which 179 were contigs. The unique sequences generated 1,056 potential open reading frames containing an average GC content of 51%. These results are consistent with previous studies on GC content within monocots and dicots [<a href="#B17" class="xref-link">17</a>].</p>
						</div>
						<div class="section"><a id="d4689524e599" class="named-anchor"><!-- named anchor --></a>
							<h5 class="section-title">Simple sequence repeats</h5>
							<p id="d4689524e602">Simple sequence repeats (or microsatellites) are a class of molecular markers that are often polymorphic and are widely used for generating genetic maps [<a href="#B20" class="xref-link">20</a>]. A total of 352 potential SSR markers were discovered within the BAC end-sequences (Table <a href="#T4" class="xref-link">4</a>). The most abundant SSRs in all four size categories were AT-rich. This is in agreement with previous reports of microsatellite abundance in other species: poly(AT)/(TA) and AT-rich trinucleotide repeats were the most abundant repeats of their class in <i>A. thaliana </i>and in yeast [<a href="#B21" class="xref-link">21</a>]. Similar to observations for <i>Rosaceae </i>ESTs [<a href="#B22" class="xref-link">22</a>], dinucleotide repeats represent the most abundant of the four microsatellite classes. None of the SSRs present in this study has been reported previously and no matches were found with previous identified Musa SSRs [<a href="#B23" class="xref-link">23</a>,<a href="#B24" class="xref-link">24</a>].</p>
							<div class="table-wrap panel"><a id="T4" class="named-anchor"><!-- named anchor --></a>
								<div class="panel-content">
									<div class="label">Table&nbsp;4</div>
									<div class="caption">
										<p class="first" id="d4689524e633">Distribution of SSRs</p>
									</div>
									<div class="table-container">
										<table class="table-figure">
											<thead>
												<tr>
													<td rowspan="1" colspan="1">Repeats</td>
													<td rowspan="1" colspan="1">Total</td>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td rowspan="1" colspan="1">A/T</td>
													<td rowspan="1" colspan="1">4</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">C/G</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AC/GT</td>
													<td rowspan="1" colspan="1">16</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AG/CT</td>
													<td rowspan="1" colspan="1">111</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AT/AT</td>
													<td rowspan="1" colspan="1">130</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AAC/GTT</td>
													<td rowspan="1" colspan="1">2</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AAG/CTT</td>
													<td rowspan="1" colspan="1">21</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AAT/ATT</td>
													<td rowspan="1" colspan="1">18</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">ACC/GGT</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">ACG/CTG</td>
													<td rowspan="1" colspan="1">3</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">ACT/ATG</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AGG/CCT</td>
													<td rowspan="1" colspan="1">4</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AGT/ATC</td>
													<td rowspan="1" colspan="1">3</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">CCG/CGG</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AAAC/GTTT</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AAAG/CTTT</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AAAT/ATTT</td>
													<td rowspan="1" colspan="1">12</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AACG/CTTG</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AACT/ATTG</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AATC/AGTT</td>
													<td rowspan="1" colspan="1">2</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AATG/ACTT</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">ACAT/ATGT</td>
													<td rowspan="1" colspan="1">4</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">ACCT/ATGG</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">ACGC/CGTG</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">ACGT/ATGC</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AGAT/ATCT</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AGCG/CGCT</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AACCG/CTTGG</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AATAT/ATATT</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AATGG/ACCTT</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AGAGC/CGTCT</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">ACATAT/ATATGT</td>
													<td rowspan="1" colspan="1">2</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">ACTCGG/AGCCTG</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">AGATAT/ATATCT</td>
													<td rowspan="1" colspan="1">1</td>
												</tr>
											</tbody>
										</table>
									</div>
								</div>
							</div>
						</div>
						<div class="section"><a id="d4689524e816" class="named-anchor"><!-- named anchor --></a>
							<h5 class="section-title"><i>Musa </i>BAC end tiling on the <i>O. sativa </i>and <i>A. thaliana </i>genome</h5>
							<p id="d4689524e827">For a relatively uncharacterized species where there may be synteny with some chromosomal regions of well sequenced model species, high throughput BAC end sequencing offers the potential to 'tile' the genome of the uncharacterized species onto to that of the sequenced species. BES mapping to <i>O. sativa </i>and <i>A. thaliana </i>were carried out in order to further characterize our BAC library and to test whether a BAC end sequencing approach might be effective for <i>Musa </i>in the manner described above. When the Musa BESs were compared to <i>O. sativa </i>genome sequence (TIGR <i>O. sativa </i>assembly version 4.0 [<a href="#B15" class="xref-link">15</a>]), 2,646 had a significant hit to <i>O. sativa </i>with percent identities ranging from 58% – 98% for top matches. These hits included 593 paired reads of which a total of 55 pairs were shown to have the top blast hit to the same chromosome after filtering for homology to mitochondrial and chloroplast matches. Eight BES pairs were shown to have similarity matches of <i>O. sativa </i>sequence with a span of 100 to 500 Kb (Table <a href="#T5" class="xref-link">5</a>). When the Musa BESs were compared to <i>A. thaliana genome</i>[<a href="#B7" class="xref-link">7</a>], 2,177 had matches, with percent identities ranging from 54% – 98% for top matches. Amongst the 2,177 hits, 403 BES pairs had a significant BLAST match (both members of the pair) to <i>A. thaliana </i>genome sequence of which a total of 36 pairs were shown to have the top blast hit to the same chromosome after filtering for homology to mitochondria and chloroplast matches. Although a small number of BES pairs were shown to have similarity matches of <i>A. thaliana </i>sequence with a span of 22 to 500 kb none of them were found in the proper orientation which may represent localised inversions.</p>
							<div class="table-wrap panel"><a id="T5" class="named-anchor"><!-- named anchor --></a>
								<div class="panel-content">
									<div class="label">Table&nbsp;5</div>
									<div class="caption">
										<p class="first" id="d4689524e874">Musa BAC end tiling on the O. sativa genome</p>
									</div>
									<div class="table-container">
										<table class="table-figure">
											<thead>
												<tr>
													<td rowspan="1" colspan="1">Reads</td>
													<td rowspan="1" colspan="1">Clone</td>
													<td rowspan="1" colspan="1">Coordinates (bp)</td>
													<td rowspan="1" colspan="1">Span (bp)</td>
													<td rowspan="1" colspan="1"><i>O. sativa </i>chromosomal location</td>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td rowspan="1" colspan="1">MAMAC34TF/MAMAC34TR</td>
													<td rowspan="1" colspan="1">MAMAC34</td>
													<td rowspan="1" colspan="1">8780081-9289856</td>
													<td rowspan="1" colspan="1">509,775</td>
													<td rowspan="1" colspan="1">4</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">MAMA945TF/MAMA945TR</td>
													<td rowspan="1" colspan="1">MAMA945</td>
													<td rowspan="1" colspan="1">25025509-24588294</td>
													<td rowspan="1" colspan="1">437,215</td>
													<td rowspan="1" colspan="1">8</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">MAMAH84TF/MAMAH84TR</td>
													<td rowspan="1" colspan="1">MAMAH84</td>
													<td rowspan="1" colspan="1">2641587-2209898</td>
													<td rowspan="1" colspan="1">431,689</td>
													<td rowspan="1" colspan="1">2</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">MAMAE66TF/MAMAE66TR</td>
													<td rowspan="1" colspan="1">MAMAE66</td>
													<td rowspan="1" colspan="1">19669753-19399800</td>
													<td rowspan="1" colspan="1">269,953</td>
													<td rowspan="1" colspan="1">10</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">MAMA777TF/MAMA777TR</td>
													<td rowspan="1" colspan="1">MAMA777</td>
													<td rowspan="1" colspan="1">20538799-20725362</td>
													<td rowspan="1" colspan="1">186,563</td>
													<td rowspan="1" colspan="1">8</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">MAMA481TF/MAMA481TR</td>
													<td rowspan="1" colspan="1">MAMA481</td>
													<td rowspan="1" colspan="1">23108313-22926983</td>
													<td rowspan="1" colspan="1">181,330</td>
													<td rowspan="1" colspan="1">5</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">MAMAZ34TF/MAMAZ34TR</td>
													<td rowspan="1" colspan="1">MAMAZ34</td>
													<td rowspan="1" colspan="1">30878620-30754915</td>
													<td rowspan="1" colspan="1">123,705</td>
													<td rowspan="1" colspan="1">3</td>
												</tr>
												<tr>
													<td rowspan="1" colspan="1">MAMAA26TF/MAMAA26TR</td>
													<td rowspan="1" colspan="1">MAMAA26</td>
													<td rowspan="1" colspan="1">34290570-34168654</td>
													<td rowspan="1" colspan="1">121,916</td>
													<td rowspan="1" colspan="1">4</td>
												</tr>
											</tbody>
										</table>
									</div>
								</div>
							</div>
							<p id="d4689524e980"><i>Musa </i>BACs that fulfil the criteria of having top blast hits to the same chromosome and having no homology to mitochondria and chloroplast were deemed candidate putative comparative-tile-BACs, and potentially represent regions of highly conserved gene content and organization. The predicted size of the <i>Musa </i>BACs (and thus the distance between the end-sequences) was compared to the span by which the paired matches are separated in the <i>O. sativa </i>and <i>A. thaliana </i>genomes respectively. Separations in the <i>Musa </i>BES matches that exceeded our arbitrary cut off of 500 Kb, may represent expansions of the syntenic regions and due to rearrangements during the evolution of the two genomes.</p>
						</div>
					</div>
					<div class="section"><a id="d4689524e996" class="named-anchor"><!-- named anchor --></a>
						<h4 class="main-title">Conclusion</h4>
						<p id="d4689524e999">In this study, 2 major ideas were examined. Firstly, that Musa BES can lead to insights into the Musa genome with specific reference to gene density, GC content, protein and SSR discovery; and secondly, that the sequences can be used to identify regions of potential microsynteny between Musa and other species. The BAC end-sequences were shown to contain homology to proteins, expressed sequence tags, transposons, repeat sequences and to be useful for simple sequence repeat identification and estimation of gene statistics and GC content. Proteins encoded in these BES were shown to cover a broad range of GO categories. Although there is only limited microsynteny between Musa and O. sativa, the results suggest that a large-scale BAC end sequencing strategy has the potential to anchor at least a small portion of the genome of Musa onto that of the sequence of the O. sativa. Large-scale BAC end sequencing would show whether there are more regions of microsynteny between the reference genome and the genome of interest and if there was support for whole genome sequencing due to unique gene features and genome characteristics. BAC end data would be one useful indicator along with existing EST or genomic sequences for funding bodies to use when selecting new plant genomes to sequence and assess the potential of leveraging the finished genomes of A. thaliana and O. sativa through comparative genomics. We expect that a similar analysis using other plant or animal species would provide insights into the genome in a very cost effective and efficient manner through database searches and synteny to model species.</p>
					</div>
					<div class="section"><a id="d4689524e1001" class="named-anchor"><!-- named anchor --></a>
						<h4 class="main-title">Methods</h4>
						<div class="section"><a id="d4689524e1004" class="named-anchor"><!-- named anchor --></a>
							<h5 class="section-title">BAC end sequencing</h5>
							<p id="d4689524e1007">The BES were generated from a <i>Musa </i>bacterial artificial chromosome (BAC) library constructed from leaves of the wild diploid 'Calcutta 4' clone (<i>Musa acuminata </i>subsp. Burmannicoides 2n = 2 × = 22) with an average insert size of 100 kb [<a href="#B12" class="xref-link">12</a>].</p>
							<p id="d4689524e1018">DNA template was prepared in 384-well format by a standard alkaline lysis method. End sequencing was performed using Applied Biosystems (ABI) Big Dye terminator chemistry and analyzed on ABI 3730 xl machines. Base calling was performed using TraceTuner and sequences were trimmed for vector and low quality sequences using Lucy [<a href="#B25" class="xref-link">25</a>].</p>
						</div>
						<div class="section"><a id="d4689524e1023" class="named-anchor"><!-- named anchor --></a>
							<h5 class="section-title">BAC end database searches</h5>
							<p id="d4689524e1026">Sequences were compared to all entries in the TIGR Plant Gene Indices [<a href="#B26" class="xref-link">26</a>] using blat and to the TIGR non-identical amino acids database that contains non-identical protein data from a number of databases including GenBank, RefSeq and Uniprot using blastx (cut-off value 1e-5). The BAC end-sequences were also compared with repetitive sequences in the TIGR Repeat Database [<a href="#B27" class="xref-link">27</a>] and an in-house transposon database using blastx with a cut-off value of 1e-5. The BAC end-sequences were compared with the TIGR rice genome sequence assembly and the <i>A. thaliana </i>genome sequence from TAIR using blastn with a cut-off value of 1e-10. To identify comparative tile BACs from the <i>Musa </i>library that were likely collinear (i.e. showed microsynteny) with the reference genomes, the searches against the <i>Musa </i>genomic sequence were parsed for the top pair of BES for which both ends had the highest significant match to a stretch of <i>O. sativa </i>or <i>A. thaliana </i>sequence and where the two regions on the <i>Musa </i>genome were between 100 kb and 500 Kb apart. The BAC end data sets for <i>O. sativa, A. thaliana</i>, maize and <i>M. truncatula </i>used for GC profiling was originally downloaded from GenBank and then the vector trimmed and cleaned sequences were downloaded from estinformatics.org [<a href="#B28" class="xref-link">28</a>].</p>
						</div>
						<div class="section"><a id="d4689524e1062" class="named-anchor"><!-- named anchor --></a>
							<h5 class="section-title">EST clustering and assembly</h5>
							<p id="d4689524e1065"><i>Musa </i>EST reads was originally downloaded from GenBank and then the vector trimmed and cleaned sequences were downloaded from estinformatics.org [<a href="#B28" class="xref-link">28</a>] and clustered and assembled [<a href="#B26" class="xref-link">26</a>].</p>
						</div>
						<div class="section"><a id="d4689524e1075" class="named-anchor"><!-- named anchor --></a>
							<h5 class="section-title">Identification and analyses of simple sequence repeats</h5>
							<p id="d4689524e1078">Perfect dinucleotide to hexanucleotide simple sequence repeats were identified using the MISA [<a href="#B20" class="xref-link">20</a>] Perl scripts, specifying a minimum of six dinucleotide and five tetranucleotide to hexanucleotide repeats and a maximum of 100-nucleotides interruption for compound repeats and the minimum length for mononucleotide repeats was 20 bases.</p>
						</div>
					</div>
					<div class="section"><a id="d4689524e1083" class="named-anchor"><!-- named anchor --></a>
						<h4 class="main-title">Authors' contributions</h4>
						<p id="d4689524e1086">FC conducted the bioinformatics, FC, CDT contributed to the manuscript writing, CDT managed the overall project. Both authors read and approved the final manuscript.</p>
					</div>
				</div>
			</section>
			<section class="so-layout-section so-section-collapser" id="d4689524e1089">
				<header class="so-layout-section-header so-openable so-closed">
					<h3 class="so-layout-section-title">Acknowledgements</h3>
				</header>
				<div class="so-d">
					<div class="back-part">
						<div class="section"><a id="d4689524e1090" class="named-anchor"><!-- named anchor --></a>
							<p id="d4689524e1093">This work was supported by the International Network for the Improvement of Banana and Plantain (INIBAP), now part of Bioversity International, through agrant under theUSAID linkage fundscheme.</p>
						</div>
					</div>
				</div>
			</section>
			<section class="so-layout-section so-section-collapser" id="d4689524e1095">
				<header class="so-layout-section-header so-openable so-closed">
					<h3 class="so-layout-section-title">References</h3>
				</header>
				<div class="so-d">
					<div class="section ref-list">
						<ol class="reflist">
							<li><span class="citation so-ol" id="B1">Meinke DW, Cherry JM, Dean C, Rounsley SD, Koornneef M authors. Arabidopsis thaliana: a model plant for genome analysis. Science. 1998;Vol. 662:679–682</span></li>
							<li><span class="citation so-ol" id="B2">International Rice Genome Sequencing Project author. The map based sequence of the rice genome. Nature. 2005;Vol. 436:793–800. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/16100779" data-untrusted="" id="d4689524e1102">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1038/nature03895" data-untrusted="" id="d4689524e1105">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B3">Zhao W, Wang J, He X, Huang X, Jiao Y, Dai M, Wei S, Fu J, Chen Y, Ren X, Zhang Y, Ni P, Zhang J, Li S, Wang J, Wong GK, Zhao H, Yu J, Yang H, Wang J authors. BGI-RIS, An integrated information resource and comparative analysis workbench for rice genomics. Nucleic Acids Res. 2004;Vol. 32:D377–82. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/14681438" data-untrusted="" id="d4689524e1111">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1093/nar/gkh085" data-untrusted="" id="d4689524e1114">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B4">Lysak MA, Dolezelova M, Horry JP, Swennen R, Dolezel J authors. Flow cytometric analysis of nuclear DNA content in Musa. Theor Appl Genet. 1999;Vol. 98:1344–1350. [<a target="xrefwindow" href="http://dx.doi.org/10.1007/s001220051201" data-untrusted="" id="d4689524e1120">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B5">Kamate K, Brown S, Durand P, Bureau JM, De Nay D, Trinh TH authors. Nuclear DNA content and base composition in 28 taxa of Musa. Genome. 2001;Vol. 44:622–627. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/11550896" data-untrusted="" id="d4689524e1126">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1139/gen-44-4-622" data-untrusted="" id="d4689524e1129">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B6">Bartos J, Alkhimova O, Dolezelova M, De Langhe E, Dolezel  authors. Nuclear genome size and genomic distribution of ribosomal DNA in Musa and Ensete (Musaceae): taxonomic implications. Cytogenet Genome Res. 2005;Vol. 109:50–7. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/15753558" data-untrusted="" id="d4689524e1135">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1159/000082381" data-untrusted="" id="d4689524e1138">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B7">Arabidopsis Genome Initiative author. Analysis of the genome sequence of the flowering plant Arabidopsis thaliana. Nature. 2000;Vol. 408:796–815. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" data-untrusted="" id="d4689524e1144">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1038/35048692" data-untrusted="" id="d4689524e1147">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B8">Singh NK, Raghuvanshi S, Srivastava SK, Gaur A, Pal AK, Dalal V, Singh A, Ghazi IA, Bhargav A, Yadav M, Dixit A, Batra K, Gaikwad K, Sharma TR, Mohanty A, Bharti AK, Kapur A, Gupta V, Kumar D, Vij S, Vydianathan R, Khurana P, Sharma S, McCombie WR, Messing J, Wing R, Sasaki T, Khurana P, Mohapatra T, Khurana JP, Tyagi AK authors. Sequence analysis of the long arm of rice chromosome 11 for rice-wheat synteny. Funct Integr Genomics. 2004;Vol. 4:102–17. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/15085449" data-untrusted="" id="d4689524e1153">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1007/s10142-004-0109-y" data-untrusted="" id="d4689524e1156">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B9">Gu Y, Coleman-Derr D, Kong X, Anderson O authors. Rapid genome evolution revealed by comparative sequence analysis of orthologous regions from four triticeae genomes. Plant Physiol. 2004;Vol. 135:459–470. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/15122014" data-untrusted="" id="d4689524e1162">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1104/pp.103.038083" data-untrusted="" id="d4689524e1165">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B10">Salse J, Piegu B, Cooke R, Delseny M authors. New in silico insight into the synteny between rice (Oryza sativa L.) and maize (Zea mays L.) highlights reshuffling and identifies new duplications in the rice genome. Plant J. 2004;Vol. 38:396–409. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/15086801" data-untrusted="" id="d4689524e1171">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1111/j.1365-313X.2004.02058.x" data-untrusted="" id="d4689524e1174">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B11">Lai CW, Yu Q, Hou S, Skelton RL, Jones MR, Lewis KL, Murray J, Eustice M, Guan P, Agbayani R, Moore PH, Ming R, Presting GG authors. Analysis of papaya BAC end sequences reveals first insights into the organization of a fruit tree genome. Mol Genet Genomics. 2006;Vol. 276:1–12. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/16703363" data-untrusted="" id="d4689524e1180">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1007/s00438-006-0122-z" data-untrusted="" id="d4689524e1183">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B12">Vilarinhos AD, Piffanelli P, Lagoda P, Thibivilliers S, Sabau X, Carreel F, D'Hont A authors. Construction and characterization of a bacterial artificial chromosome library of banana (Musa acuminata Colla). Theor Appl Genet. 2003;Vol. 106:1102–6. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/12671759" data-untrusted="" id="d4689524e1190">PubMed</a>]</span></li>
							<li><span class="citation so-ol" id="B13">SanMiguel P, Gaut BS, Tikhonov A, Nakajima Y, Bennetzen JL authors. The paleontology of intergene retrotransposons of maize. Nat Genet. 1998;Vol. 20:43–5. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/9731528" data-untrusted="" id="d4689524e1196">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1038/1695" data-untrusted="" id="d4689524e1199">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B14">Aert R, Sagi L, Volckaert G authors. Gene content and density in banana (Musa acuminata) as revealed by genomic sequencing of BAC clones. Theor Appl Genet. 2004;Vol. 109:129–139. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/14985976" data-untrusted="" id="d4689524e1205">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1007/s00122-004-1603-2" data-untrusted="" id="d4689524e1208">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B15">Yuan Q, Ouyang S, Wang A, Zhu W, Maiti R, Lin H, Hamilton J, Haas B, Sultana R, Cheung F, Wortman J, Buell CR authors. The Institute for Genomic Research Osa1 rice genome annotation database. Plant Physiol. 2005;Vol. 138:18–26. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/15888674" data-untrusted="" id="d4689524e1214">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1104/pp.104.059063" data-untrusted="" id="d4689524e1217">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B16">The Arabidoposis Information Resource. <a target="xrefwindow" href="http://www.arabidopsis.org" data-untrusted="" id="d4689524e1223">http://www.arabidopsis.org</a></span></li>
							<li><span class="citation so-ol" id="B17">Kuhl JC, Cheung F, Yuan Q, Martin W, Zewdie Y, McCallum J, Catanach A, Rutherford P, Sink KC, Jenderek M, Prince JP, Town CD, Havey MJ authors. A unique set of 11,008 onion expressed sequence tags reveals expressed sequence and genomic differences between the monocot orders Asparagales and Poales. Plant Cell. 2004;Vol. 16:114–25. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/14671025" data-untrusted="" id="d4689524e1227">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1105/tpc.017202" data-untrusted="" id="d4689524e1230">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B18">Wong GK, Wang J, Tao L, Tan J, Zhang J, Passey DA, Yu J authors. Compositional gradients in Gramineae genes. Genome Res. 2002;Vol. 12:851–856. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/12045139" data-untrusted="" id="d4689524e1236">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1101/gr.189102" data-untrusted="" id="d4689524e1239">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B19">Santos CM, Martins NF, Horberg HM, de Almeida ER, Coelho MC, Togawa RC, da Silva FR, Caetano AR, Miller RN, Souza MT Jr authors. Analysis of expressed sequence tags from Musa acuminata ssp burmannicoides, var. Calcutta 4 (AA) leaves submitted to temperature stresses. Theor Appl Genet. 2005;Vol. 110:1517–1522. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/15841358" data-untrusted="" id="d4689524e1245">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1007/s00122-005-1989-5" data-untrusted="" id="d4689524e1248">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B20">Thiel T, Michalek W, Varshney RK, Graner A authors. Exploiting EST databases for the development and characterization of gene-derived SSR-markers in barley (Hordeum vulgare L.). Theor Appl Genet. 2003;Vol. 106:411–422. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/12589540" data-untrusted="" id="d4689524e1254">PubMed</a>]</span></li>
							<li><span class="citation so-ol" id="B21">Katti MV, Ranjekar PK, Gupta VS authors. Differential distribution of simple sequence repeats in eukaryotic genome sequences. Mol Biol Evol. 2001;Vol. 18:1161–7. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/11420357" data-untrusted="" id="d4689524e1260">PubMed</a>]</span></li>
							<li><span class="citation so-ol" id="B22">Jung S, Abbott A, Jesudurai C, Tomkins J, Main D authors. Frequency, type, distribution and annotation of simple sequence repeats in Rosaceae ESTs. Funct Integr Genomics. 2005;Vol. 5:136–43. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/15761705" data-untrusted="" id="d4689524e1266">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1007/s10142-005-0139-0" data-untrusted="" id="d4689524e1269">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B23">Creste S, Benatti TR, Orsi MR, Risterucci AM, Figueira A authors. Isolation and characterization of microsatellite loci from a commercial cultivar of <i>Musa acuminata</i>. Molecular Ecology Notes. 2006;Vol. 6:303–306. [<a target="xrefwindow" href="http://dx.doi.org/10.1111/j.1471-8286.2005.01209.x" data-untrusted="" id="d4689524e1279">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B24">Raboin LM, Carreel F, Noyer JL, Baurens FC, Horry JP, Bakry F, Tezenas Du Montcel H, Ganry J, Lanaud C, Lagoda PJL authors. Diploid ancestors of triploid export banana cultivars: molecular identification of 2n restitution gamete donors and n gamete donors. Molecular Breeding. 2005;Vol. 16:333–341. [<a target="xrefwindow" href="http://dx.doi.org/10.1007/s11032-005-2452-7" data-untrusted="" id="d4689524e1285">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B25">Chou HH, Holmes MH authors. DNA sequence quality trimming and vector removal. Bioinformatics. 2001;Vol. 17:1093–1104. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/11751217" data-untrusted="" id="d4689524e1291">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1093/bioinformatics/17.12.1093" data-untrusted="" id="d4689524e1294">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B26">Quackenbush J, Liang F, Holt I, Pertea G, Upton J authors. The TIGR gene indices: reconstruction and representation of expressed gene sequences. Nucleic Acids Res. 2000;Vol. 28:141–145. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/10592205" data-untrusted="" id="d4689524e1300">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1093/nar/28.1.141" data-untrusted="" id="d4689524e1303">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B27">Ouyang S, Buell CR authors. The TIGR Plant Repeat Databases: a collective resource for the identification of repetitive sequences in plants. Nucleic Acids Res. 2004;Vol. 32:D360–3. [<a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/14681434" data-untrusted="" id="d4689524e1309">PubMed</a>] [<a target="xrefwindow" href="http://dx.doi.org/10.1093/nar/gkh099" data-untrusted="" id="d4689524e1312">Cross Ref</a>]</span></li>
							<li><span class="citation so-ol" id="B28">estinformatics.org. <a target="xrefwindow" href="http://www.estinformatics.org" data-untrusted="" id="d4689524e1318">http://www.estinformatics.org</a></span></li>
						</ol>
					</div>
				</div>
			</section>
		</div>
	</body>
</html>